source/opt/def_use_manager.h

```diff
@@ -95,34 +94,31 @@ class DefUseManager {
   bool ReplaceAllUsesWith(uint32_t before, uint32_t after);
 
  private:
+  using InstToUsedIdsMap =
+      std::unordered_map<const ir::Instruction*, std::vector<uint32_t>>;
+
   // Analyzes the defs and uses in the given |module| and populates data
   // structures in this class. Does nothing if |module| is nullptr.
   void AnalyzeDefUse(ir::Module* module);
 
-  // Clear the internal def-use record of a defined id if the given |def_id| is
-  // recorded by this manager. This method will erase both the uses of |def_id|
-  // and the |def_id|-generating instruction's use information kept in this
-  // manager, but not the operands in the original instructions. Does nothing
-  // if |def_id| was not recorded before.
-  void ClearDef(uint32_t def_id);
-
   // Clear the internal def-use record of the given instruction |inst|. This
   // method will update the use information of the operand ids of |inst|. The
   // record: |inst| uses an |id|, will be removed from the use records of |id|.
   // If |inst| defines an result id, the use record of this result id will also
   // be removed. Does nothing if |inst| was not analyzed before.
   void ClearInst(ir::Instruction* inst);
 
-  // Erases the records that a given instruction |inst| uses its operand ids.
+  // Erases the records that a given instruction uses its operand ids.
```

Comment:
Generally we want to avoid mentioning implementation details in function blurbs. I feel that mentioning updates of `id_to_uses_` & `id_to_used_ids_` should be avoided. I think the name is fairly self-explaining and the original blurb is actually good.